c172p-fdm-dany93.xml

```diff
@@ -939,13 +950,23 @@
                     <value>0.0147</value>
                 </product>
             </function>
-            <function name="aero/coefficient/Clbk">
-                <description>Roll_moment_due_to_one_broken wing</description>
+            <function name="aero/coefficient/Cllwdmg">
```

Comment:
They are naturally summed like every coefficient which is inside an ```<axis>``` (ROLL, here) channel.
They are used and set in physics.nas via the properties
/fdm/jsbsim/wing-damage/left-wing
/fdm/jsbsim/wing-damage/right-wing
which take the values between 0 (no damage) and 1 (totally broken).